Wrappers.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include "Ipad.h"
+#include "Housing.h"
+#include "MobilePhone.h"
+#include "WorkEnvironment.h"
+#include "StudyEnvironment.h"
```

Comment:
Forward declare all of these types.  You aren't using any of them in this header file, but you are in the cpp file.
- [ ] revise